chronos/asyncloop.nim

```diff
@@ -232,9 +233,17 @@ template processTimersGetTimeout(loop, timeout: untyped) =
   if count > 0:
     var lastFinish = curTime
     while count > 0:
+      if not(loop.timers.len > 0):
```

Comment:
why not
```nim
if loop.timers.len == 0:
  break
```
?

Reply:
The `count` check above is essentially doing the same thing, but it "optimised" way. I would just remove it altogether and make the loop

```nim
while loop.timers.len > 0:
  ...
```

Reply:
Not sure if another while is going to add much for readability? Not opposed, it just seems to be more of a style thing?

Reply:
I've meant merging the while and the if, not introducing another while.

Reply:
Totally agree with @zah proposition to replace
```nim
while count > 0:
```
with 
```nim
while loop.timers.len > 0:
```

and remove `if`